en/README.md

```diff
@@ -0,0 +1,1138 @@
+    |                                              |
+    |<-----------getLastInfo(Request)--------------|
+    |------------getLastInfo(Response)------------>|
+    |<-----putHostGroupMembership(Request)---------|
+    |------putHostGroupMembership(Response)------->|
+    |                                              |
+    |<-----------getLastInfo(Request)--------------|
+    |------------getLastInfo(Response)------------>|
+    |<------------putTriggers(Request)-------------|
+    |-------------putTriggers(Response)----------->|
+    |                                              |
+    |<-----------getLastInfo(Request)--------------|
+    |------------getLastInfo(Response)------------>|
+    |<------------putEvents(Request)---------------|
+    |-------------putEvents(Response)------------->|
+    |                                              |
+    |<-----------getLastInfo(Request)--------------|
+    |------------getLastInfo(Response)------------>|
+    |<------------putHostParents(Request)----------|
+    |-------------putHostParents(Response)-------->|
+    |                                              |
+    |<------------putArmInfo(Request)--------------|
+    |-------------putArmInfo(Response)------------>|
+    |                                          |   |
+    |                             Polling interval |
+    |                                          |   |
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    |                                              |
+    |-----------fetchItems(Request)--------------->|
+    |<----------fetchItems(Response)---------------|
+    |<------------putItems(Request)----------------|
+    |-------------putItems(Response)-------------->|
+    |                                              |
+    |-----------fetchHistory(Request)------------->|
+    |<----------fetchHistory(Response)-------------|
+    |<------------putHistory(Request)--------------|
+    |-------------putHistory(Response)------------>|
+    |                                              |
+    |-----------fetchTriggers(Request)------------>|
+    |<----------fetchTriggers(Response)------------|
+    |<-----------putTriggers(Request)--------------|
+    |                Use "ALL"option               |
+    |-----------putTriggers(Response)------------->|
+    |                                              |
+    |-------------fetchEvents(Request)------------>|
+    |<------------fetchEvents(Response)------------|
+    |<------------putEvents(Request)---------------|
+    |-------------putEvents(Response)------------->|
+    |                                              |
+    |--updateMonitoringServerInfo(notification)--->|
+    |                                              |
+
+```
+## Data type
+
+ - In this section explains about data type that is defined for Hatohol. They use date type that is defined in the JSON-RPC internally.
```

Comment:
`data type` can be plural form.
So, this sentence becomes as follows:
In this section explains about data type**s** that **are** defined for Hatohol.